Dependencies

e-mp opened the pull request.

Commit:
Front-End Dependencies Added

Commit:
Renamed Folder

Commit:
added start scripts

KingAtoki reviewed (changes requested):
Make sure to .gitignore `node_modules` and commit to github

Commit:
removed package.json and node_modules out of the root directory
added necessary dependencies and dev dependencies to FrontEnd package.json
corrected typo

Commit:
re-named folder for the last time hopefully

KingAtoki reviewed (approved):
All good now!
